src/app/components/core/north/north.component.ts

```diff
@@ -107,11 +107,5 @@ export class NorthComponent implements OnInit, OnDestroy {
   public hideLoadingSpinner() {
     this.showSpinner = false;
   }
-
-  public ngOnDestroy(): void {
-    this.isAlive = false;
```

Comment:
is this also being taken care of by super?

Reply:
ngOnDestroy() revered now at child component level to fix this issue.